hermes-frontend/src/main/java/pl/allegro/tech/hermes/frontend/HermesFrontend.java

```diff
@@ -115,11 +127,12 @@ public static Builder frontend() {
         private final List<Binder> binders = Lists.newArrayList(new CommonBinder(), new FrontendBinder());
         private final BrokerListeners listeners = new BrokerListeners();
         private final List<Function<ServiceLocator, LogRepository>> logRepositories = new ArrayList<>();
+        private Optional<Function<ServiceLocator, KafkaNamesMapper>> kafkaNamesMapper = Optional.empty();
```

Comment:
I don't like that we expose directly hk2 ServiceLocator in our API. Unfortunately I don't have a better solution for it at the moment. I must rethink this feature.